_harp/academy/_faq.md

```diff
@@ -40,7 +43,7 @@ This programme is challenging and you need to be well-prepared. We aim for a zer
 Very. We get many applicants for each place.
 
 **How much does it cost?**     
-There are no fees, but students and graduates are expected to help us deliver the course to the next cohort and to contribute to our various community projects.
+There are no fees, but students and graduates are expected to help us deliver the course to the next cohort and to contribute to our various community projects. You will also need to cover your own living expenses for at least 4 months, and quite possibly longer, depending on how quickly you can find work. 
```

Comment:
Just being picky but I would say 'for at least the 16 weeks of the course' instead of 4 months to make the link to the course explicit.